.dockerignore

```diff
@@ -2,13 +2,14 @@
 **/*
 
 # base
-!/tools/install-deps.sh
+!/vtools/scripts/install-deps.sh
```

Comment:
@0xdiba i think this should really be a single command `vtools scripts install-deps.sh`